DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/AddRegistrationPromptSummary.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+@using Microsoft.Extensions.Configuration
+@model AddRegistrationPromptSummaryViewModel
+
+@{
+  ViewData["Title"] = "Add delegate registration prompt";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["CurrentSystemBaseUrl"]}/tracking/dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-u-margin-bottom-8">
+      <h1 id="page-heading" class="nhsuk-heading-xl">Summary</h1>
+    </div>
+
+    <dl class="nhsuk-summary-list">
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Prompt
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.PromptName
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="AddRegistrationPromptSelectPrompt">
+            Change<span class="nhsuk-u-visually-hidden"> prompt</span>
+          </a>
+        </dd>
+      </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Is mandatory?
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.Mandatory
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="AddRegistrationPromptSelectPrompt">
+            Change<span class="nhsuk-u-visually-hidden"> whether this prompt is mandatory</span>
+          </a>
+        </dd>
+      </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Answers
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @if (Model.Answers.Any()) {
+            <ul>
+              @foreach (var answer in Model.Answers)
+              {
+                <li class="word-break">@answer</li>
+              }
+            </ul>
+          } else {
+            <partial name="_NoConfiguredAnswers"/>

```

Comment:
Should we just leave it empty in the summary or maybe write something fairly short here given that it's a summary? Do you have any screenshots of the current version?

Reply:
![image](https://user-images.githubusercontent.com/59561751/120768794-6e91d180-c514-11eb-87b5-87572827e21c.png)

Reply:
Oh actually it doesn't look bad so let's leave it as is! 👍  Thanks for the screenshot! :)